### PR TITLE
added color widget (difference to the color button, is thats its not hidden behind a button so you can always see it.

### PR DESCRIPTION
**Pull request by alx365**
The bug still exists with printing the variable, so i can't really test what the color value is.. it probably is correct. I hope so at least

### CODE REVIEW

**Comment by elkowar** [comment]
I'll wait until #37 is fixed, which should hopefully be this evening ^^ thanks for your contributions!

File: src/widgets/widget_definitions.rs [hunk 1]
```diff
@@ -22,6 +22,7 @@ pub(super) fn widget_to_gtk_widget(bargs: &mut BuilderArgs) -> Result<Option<gtk
         "calendar" => build_gtk_calendar(bargs)?.upcast(),
         "color-button" => build_gtk_color_button(bargs)?.upcast(),
         "expander" => build_gtk_expander(bargs)?.upcast(),
+        "color-widget" => build_gtk_color_chooser(bargs)?.upcast(),
```
Comment (elkowar) [inline src/widgets/widget_definitions.rs:25]:
```suggestion
        "color-chooser" => build_gtk_color_chooser(bargs)?.upcast(),
```

File: src/widgets/widget_definitions.rs [hunk 2]
```diff
@@ -178,6 +179,28 @@ fn build_gtk_color_button(bargs: &mut BuilderArgs) -> Result<gtk::ColorButton> {
     Ok(gtk_widget)
 }
 
+/// @widget color chooser
```
Comment (elkowar) [inline src/widgets/widget_definitions.rs:182]:
```suggestion
/// @widget color-chooser
```

**Review by elkowar — CHANGES_REQUESTED** [review]
Please make sure to keep the naming and documentation consistent

**Comment by alx365** [comment]
> Please make sure to keep the naming and documentation consistent

ok

**Review by alx365 — COMMENTED** [review]
github is buggy

**Comment by elkowar** [comment]
Thonks!

**Pull request merged**